ga4gh/cli.py

```diff
@@ -1710,22 +1762,17 @@ def getParser(cls):
         cls.addRepoArgument(initParser)
         cls.addForceOption(initParser)
 
-        checkParser = addSubparser(
-            subparsers, "check", "Check to see if repo is well-formed")
-        checkParser.set_defaults(runner="check")
-        cls.addRepoArgument(checkParser)
+        verifyParser = addSubparser(
+            subparsers, "verify",
+            "Verifies the repository by examing all data files")
+        verifyParser.set_defaults(runner="verify")
+        cls.addRepoArgument(verifyParser)
 
         listParser = addSubparser(
             subparsers, "list", "List the contents of the repo")
         listParser.set_defaults(runner="list")
         cls.addRepoArgument(listParser)
 
-        destroyParser = addSubparser(
```

Comment:
Yeah, it seemed bit pointless since ``ga4gh_repo destroy repo.db == rm repo.db``. We're not storing any data inside the repo anymore, so if the user deletes the file then that's their own business.